style.css

```diff
@@ -655,3 +694,343 @@ label {
   width: 90px;
   text-align: right;
 }
+
+.link-button:hover,
+.link-button2:hover,
+.resume:hover,
+.sub-bot:hover {
+  color: blue;
+  background: wheat;
+  border: blue;
+}
+
+@media screen and (min-width: 768px) {
+  main {
+    width: 768px;
+    align-items: center;
+  }
+
+  body {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .page1 {
+    background-image: url("/img/desktop-ver.png");
+    background-repeat: no-repeat;
+    width: 1440px;
+    height: 852px;
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+  }
+
+  .envelope {
+    margin-top: -20px;
+  }
+
+  .social-media {
+    order: 1;
+    align-self: center;
+    grid-column: 2/3;
+    grid-row: 3/4;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .main-header {
+    width: 1440px;
+    height: 72px;
+  }
+
+  .desktop-main {
+    order: 2;
+  }
+
+  .desktop-header {
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    column-gap: 30px;
+    justify-content: space-around;
+    margin-right: 70px;
+    display: flex;
+    text-align: center;
+    align-self: center;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .icon {
+    display: flex;
+    flex-direction: column;
+    row-gap: 15px;
+  }
+
+  .recent-work {
+    width: 1440px;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .recent-header {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+
+  .Works {
+    padding: 0;
+    grid-template-columns: 350px 350px 369px;
+    grid-template-rows: 500px 410px 480px;
+    align-items: center;
+  }
+
+  .card {
+    padding: 0;
+    margin: 0;
+    width: 369px;
+    height: 435px;
+  }
+
+  .work-img {
+    padding: 0;
+    margin: 0;
+    grid-column: 1/3;
+    width: 684px;
+    height: 448px;
+    flex: none;
+  }
+
+  .work2 {
+    padding: 0;
+    margin: 0;
+    margin-top: -20%;
+    grid-column-start: 3;
+  }
+
+  .wtext1 {
+    padding: 0;
+    margin: 0;
+    grid-column: 2/3;
+  }
+
+  .work1 {
+    display: flex;
+  }
+
+  .line {
+    margin: 0;
+    width: 794px;
+  }
+
+  .works_header-text {
+    align-self: center;
+    display: flex;
+  }
+
+  .link-button2 {
+    display: flex;
+    align-items: center;
+    margin-bottom: 10%;
+    width: 327px;
+    height: 48px;
+    background: #ff6b00;
+    border: none;
+  }
+
+  .li {
+    margin: 0;
+  }
+
+  .About-myself {
+    padding: 0;
+    margin: 0;
+    background: #262626;
+    background-image: url("/img/about.png");
+    background-repeat: no-repeat;
+    width: 1440px;
+    height: 1195px;
+    flex-direction: column;
+    display: flex;
+    align-items: flex-start;
+  }
```

Comment:
- [ ] I can see you have put a lot of effort into designing your portfolio. But there are some more details we need to go through to make your design as close as possible to the Figma design. This includes the fixed size you used. which makes your display only appropriate for a `1440px` screen size.
- [ ] Also, As You can see your design can use more styled to align with the Figma design. This includes the display of different screen sizes, colors, and background decorations. Please make sure to add more style to better design your `Featured project` section.

### Figma design
![Screenshot 2022-12-08 192827](https://user-images.githubusercontent.com/69077061/206507918-0f2fe441-5b39-438a-98b8-e9accb647706.png)

### Your design
![Screenshot 2022-12-08 192809](https://user-images.githubusercontent.com/69077061/206507904-d7b833b7-67bb-4e10-80ba-2a2e3e584849.png)